.git2gus/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "productTag": "a1aEE000000dLNxYAM",
+  "productTagLabels": {
+    "area:generatedresponses": "a1aB0000000JglYIAS"
```

Comment:
Looks like this goes to a different tag. Maybe we could set the defaultBuild to "Backlog", and add a few more of our ProductTags here?